stacker/util.py

```diff
@@ -493,6 +493,48 @@ def get_client_region(client):
     return client._client_config.region_name
 
 
+def get_template_file_format(template_path):
```

Comment:
Can we move this methods to `stacker.blueprints.raw`, since they're only used there? At the very least, it'll make reading `RawTemplateBlueprint` easier.